Drivers/seds/SunSensor.xml

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PackageFile xmlns="http://www.ccsds.org/schema/sois/seds" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.ccsds.org/schema/sois/seds seds.xsd">
+  <Package name="com/n7space/SunSensor">
+    <DeclaredInterfaceSet>
+      <Interface level="functional" name="SunSensorInterfaceType">
+        <CommandSet>
+          <Command name="InitSunSensorCmd" mode="sync">
+            <LongDescription>Initializes afec for reading Sun sensor data</LongDescription>
+            <Argument name="afecConfig" type="com/n7space/AfecHwas/AfecHwasConfig" mode="in" />

```

Comment:
What is configured here? Pin is fixed to 8, and statup + prescaler is overwritten during init.

Reply:
For simplicity, in the beginning, I decided to hardcode it and it seems like I forgot to change it back.